lib/airbrake/passenger.rb

```diff
@@ -0,0 +1,62 @@
+require 'set'
+require 'pathname'
+
+module Airbrake
+  # Integration for Phusion Passenger
+  module Passenger
+    IntegrationException = Class.new(::Exception)
```

Comment:
I hadn't seen that you catch it within the same file. I am not sure why we need to raise it in the first place.